src/utils/formating.ts

```diff
@@ -18,3 +19,14 @@ export const contenfulDateFormated = (date: string): string => {
   });
   return formatedDate;
 };
+
+export const extractCategoriesFormated = (posts: IPost[]): string[] => {
+  const avaibleCategories = posts.map((element) => element.category);
+  const categoriesFormated = avaibleCategories.filter(function (
```

Comment:
When using array methods, like `filter()`, prefer arrow functions to define the callbacks / utility functions as they are shorter and easier to read.